examples/sdp3x_default_address/sdp3x_default_address.ino

```diff
@@ -9,17 +9,18 @@ void setup() {
   Wire.begin();
   Serial.begin(9600);
   delay(1000); // let serial console settle
-
-  int ret = sdp.init();
-  if (ret == 0) {
+  bool error = false;
+  do{
```

Comment:
The project style dictates a single space before curly braces `{`. Could you please add that?